src/ThemeProvider.tsx

```diff
@@ -158,7 +163,7 @@ function chooseColorScheme(colorMode: ColorMode, dayScheme: string, nightScheme:
 
 function applyColorScheme(theme: Theme, colorScheme: string) {
```

Comment:
Should we be explicit about the return type here?
```suggestion
function applyColorScheme(theme: Theme, colorScheme: string): [Theme, string | undefined] {
```